scripts/setup/setup-python-packages.js

```diff
@@ -56,13 +56,17 @@ export default () => new Promise(async (resolve, reject) => {
         await installPythonPackages()
       } else {
         const dotProjectPath = path.join(process.cwd(), 'bridges/python/.venv/.project')
-        const dotProjectMtime = fs.statSync(dotProjectPath).mtime
+        if (fs.existsSync(dotProjectPath)) {
```

Comment:
It happens that `bridges/python/.venv/.project` doesn't exist, so it throws an error when executing `fs.statSync`.
If this file doesn't exist, we install Python packages.